lib/fnmatch.c: Fix getenv() signature
src/getopt.c: Ditto.
stc/getopt.h: Fix getopt() signature
---

diff -ru b/lib/fnmatch.c a/lib/fnmatch.c
--- b/lib/fnmatch.c	2023-02-26 11:04:31.000000000 -0500
+++ a/lib/fnmatch.c	2025-06-17 08:16:13.705463571 -0400
@@ -121,7 +121,7 @@
    whose names are inconsistent.  */
 
 # if !defined _LIBC && !defined getenv
-extern char *getenv ();
+extern char *getenv (const char*);
 # endif
 
 # ifndef errno
diff -ru b/src/getopt.c a/src/getopt.c
--- b/src/getopt.c	2023-01-11 19:53:26.000000000 -0500
+++ a/src/getopt.c	2025-06-17 08:17:07.332616352 -0400
@@ -202,7 +202,7 @@
    whose names are inconsistent.  */
 
 #ifndef getenv
-extern char *getenv ();
+extern char *getenv (const char*);
 #endif
 
 static char *
diff -ru b/src/getopt.h a/src/getopt.h
--- b/src/getopt.h	2023-01-01 10:06:01.000000000 -0500
+++ a/src/getopt.h	2025-06-17 08:18:19.766820069 -0400
@@ -102,7 +102,7 @@
    errors, only prototype getopt for the GNU C library.  */
 extern int getopt (int argc, char *const *argv, const char *shortopts);
 #else /* not __GNU_LIBRARY__ */
-extern int getopt ();
+extern int getopt (int, char* const*, const char*);
 #endif /* __GNU_LIBRARY__ */
 extern int getopt_long (int argc, char *const *argv, const char *shortopts,
 		        const struct option *longopts, int *longind);
